lib/unifex/interface_IO.ex

```diff
@@ -29,6 +29,11 @@ defmodule Unifex.InterfaceIO do
     File.write!("#{out_base_path}.c", source)
     File.write!("#{out_base_path}.cpp", source)
 
+    Mix.shell().cmd(
+      "clang-format -style=\"{BasedOnStyle: llvm, IndentWidth: 2}\" -i " <>
+        "#{out_base_path}.h #{out_base_path}.c #{out_base_path}.cpp"
```

Comment:
It would be awesome if we could store the style in `.clang-format` file

Reply:
you mean you want the style to be configurable? this would be problematic when comparing to reference files in tests

Reply:
I wanted the style to be saved to a file so it could be applied to the files created by user, but I've just realized that it will land in `_generated` folder so it doesn't make sense